debezium-testing/debezium-testcontainers-testing/src/test/resources/config.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "inventory-connector",
+  "config": {
+    "connector.class": "io.debezium.connector.mysql.MySqlConnector",
+    "database.hostname": "192.168.99.100"
```

Comment:
this looks like a magical IP. consider using network aliases instead

Reply:
It's actually used in a test un-related to TC.

Reply:
Gotcha!